builtins/amp-ad.js

```diff
@@ -283,8 +283,17 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
-      this.element.removeChild(this.iframe_);
-      this.toggleFallback(true);
+      if (!this.fallback_) {
+        this.changeHeight(0, () => {
```

Comment:
Does the height and the `display:none` happen in the same instant?

Does this do nothing if resize is not allowed?